File: Iteration1/SourceCode/src/main/java/Identity.java
public class Identity {
    private int ID;

    public Identity(int order) {
        this.ID = createID(order);

    }

    public void ID(int ID){

    }

    public int getID() {
        return ID;
    }

    public void setID(int ID) {
        this.ID = ID;
    }
    public int createID(int order){
        return order+150110000;
    }
}

File: Iteration3/SourceCode/Identity.py
class Identity(object):

    def __init__(self, order):
        self.__ID = 0

        self.__ID = self.createID(order)


    def ID(self, ID):

        pass

    def getID(self):
        return self.__ID

    def setID(self, ID):
        self.__ID = ID
    def createID(self, order):
        return order+150110000